### PR TITLE
Refactor editable attributes

### LINKED ISSUE #18: Models reinitialized without editable_attributes
reproduce:

1. start rails server
2. check localhost, it works
3. edit any html or rb file
4. localhost shows undefined method `editable_attributes'

might be related to when the method is initialized on server start

### PR DESCRIPTION
**Pull request by simon-isler**
closes #18 

out of https://github.com/renuo/hotsheet/pull/30#issuecomment-2469049261:
I don't really like how we define the editable_attributes so far. IMO there is no need to define it on existing models. I'd prefer a module with a method that returns the editable attributes for a model. This will make it also easier to test, which is essential since this logic is where everything starts :)

### CODE REVIEW

File: lib/hotsheet/editable_attributes.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      if model_config&.included_attributes
+        model_config.included_attributes.map(&:to_s)
+      elsif model_config&.excluded_attributes
+        model.column_names - model_config.excluded_attributes.map(&:to_s)
+      else
+        model.column_names
+      end
+    end
+  end
+end
```
Comment (simon-isler) [inline lib/hotsheet/editable_attributes.rb:24]: fixes the reloading issue and makes it testable :)

File: spec/system/editable_attributes_spec.rb [hunk 2]
```diff
@@ -9,6 +9,8 @@
     before do
       visit "/hotsheet"
       click_link "Author"
+
+      wait_for_turbo
```
Comment (ignaciosy) [inline spec/system/editable_attributes_spec.rb:13]: was this really necessary? Capybara should wait by default

**Review by ignaciosy — APPROVED** [review]
very nice!

**Review by hunchr — APPROVED** [review]
nice

**Pull request merged**